### DISCUSSION TITLE
Welcome to genshin_impact_assistant Discussions!

### DISCUSSION THREAD

**Discussion by infstellar** [infstellar/genshin_impact_assistant#92] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bocchi-manGo** [comment]
下载了不懂怎么使用=-=

**Comment by UnrealSam** [comment]
还在维护吗？加qq群没通过

**Comment by UnrealSam** [comment]
人也挺少，这玩意挺不错，不知道为什么搞得人少

**Comment by wuli-wuli** [comment]
自动委托非战斗类的蒙德任务做了多少了

**Reply by infstellar** [reply]
https://github.com/infstellar/genshin_impact_assistant/blob/main/source/commission/commission_acquisition.py
